uberfire-nio2-backport/uberfire-nio2-impls/uberfire-nio2-jgit/src/test/java/org/uberfire/java/nio/fs/jgit/JGitFileSystemImplProviderBytemanTest.java

```diff
@@ -333,7 +333,7 @@ public void testFileSystemLockOnException() throws IOException, GitAPIException
         } catch (Exception e) {
             fail(e.getMessage());
         }
-        assertFalse(((Boolean) isLocked));
+        assertTrue(((Boolean) isLocked));
```

Comment:
@domhanak why you changed this? the only way to get this true is (or get it accidentally, that byteman didn't quite work) OR not set 1 MILLIS to cache eviction policy:

original:

```java
        Map<String, String> gitPreferences = getGitPreferences();
        gitPreferences.put(JGIT_CACHE_EVICT_THRESHOLD_DURATION, "1");
        gitPreferences.put(JGIT_CACHE_EVICT_THRESHOLD_TIME_UNIT, TimeUnit.MILLISECONDS.name());
        provider = new JGitFileSystemProvider(gitPreferences);
```

to:
```java
        provider = new JGitFileSystemProvider();
```

Reply:
I'm sending a PR with such fix